lib/uknetdev/include/uk/netbuf.h

```diff
@@ -520,6 +553,14 @@ static inline int uk_netbuf_header(struct uk_netbuf *head, int16_t len)
 
 	head->data   -= len;
 	head->len    += len;
+
+	if (head->flags & UK_NETBUF_F_PARTIAL_CSUM) {
+		/* When header gets removed, delete PARTIAL_CSUM flag */
+		if (unlikely((len < 0) && (-len > head->csum_start)))
```

Comment:
```suggestion
		if (unlikely((len < 0) && (len < head->csum_start)))
```
Isn't it more concise like this? That `-` might be easy to miss.

Reply:
This change is would not be the same expression anymore.
It is either
`-len > head->csum_start`
or
`len < -head->csum_start`
but in both cases one minus is staying.

Reply:
Whoops, you're right